vendor/bat-native-ads/src/bat/ads/internal/ads/serving/choose/sample_ads.h

```diff
@@ -56,6 +68,20 @@ absl::optional<T> SampleAdFromPredictors(
   return absl::nullopt;
 }
 
+inline std::vector<double> ComputeProbabilities(
```

Comment:
this function should not be inlined per c++ google code style:
https://google.github.io/styleguide/cppguide.html#Inline_Functions

Reply:
clang was complaining that is should, that's why I added it

Reply:
it actually gives me a build error (linker related) if I remove it

Reply:
so you need to put function definition to .cc file